eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -185,7 +185,22 @@ jobs:
           env:
             REQUIREMENTS: $(requirements)
```

Comment:
This doesn't need to be passed in as an environment variable it can just be a standard devops variable inside the script block (i.e. `$(requirements)`).

Reply:
That should probably be updated across this entire yml file.